### DISCUSSION TITLE
Is the Deta database still a self standing service

### DISCUSSION THREAD

**Discussion by jordancrubin** [deta/cloud-docs#756] [General] [answered]
I'm reading about this deta space.  Having just finished a library for ESP32 to integrate the database at deta.  Is this all now changing?

**Comment by xeust** [comment, marked as answer]
Hi there @jordancrubin ,

You can still connect to Base exactly like before using a "Data Key" (formerly called Project Key). 

It just now lives in an app called "Collections", which you can read more about here:

https://deta.space/manual/features/collections

**Reply by jordancrubin** [reply]
Thanks for the reply, I was able to migrate.  I saw that there is now a data cap at 250mb.   Given this new limitation, for databases capturing analytical data, is there a mechanism once it should become full to purge the oldest records or will I be writing extra code to support monitoring it.   To that end, is there a means in the API to check the data utilization.
